tests/integration_tests/config/validate_cumulative_psf_prod6_psf.yml

```diff
@@ -8,6 +8,6 @@ CTA_SIMPIPE:
     DATA: ./tests/resources/PSFcurve_data_v2.txt
     ZENITH: 20
     TEST: true
-    OUTPUT_PATH: simtools-tests
+    OUTPUT_PATH: simtools-output
   INTEGRATION_TESTS:
-    - OUTPUT_FILE: ray-tracing/results/ray-tracing-North-LSTN-01-d10.0km-za20.0deg_validate_cumulative_psf.ecsv
+    - OUTPUT_FILE: results/ray-tracing-North-LSTN-01-d10.0km-za20.0deg_validate_cumulative_psf.ecsv
```

Comment:
Just want to make sure `results` is intended here in the output file name? ray-tracing would be `sub_dir`. Is this correct? `results` appears also in several other places.

Reply:
Yes, this was changed to make it consistent with the other tests.